parallelization_test/parallelization-benchmark.ipynb

```diff
@@ -0,0 +1,1325 @@
+{
```

Comment:
<p>the way it is set up now both fetches are run even when we are not interested in the processing time as <code>benchmark_result.csv</code> exists. How about moving the fetching data into the condition if the file does not exist already (below <code>if not os.path.isfile('benchmark_result.csv'):</code> ) ?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bokulich-lab/q2-fondue-examples/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>